client_test.go

```diff
@@ -129,7 +129,21 @@ func TestClient_GetOutputConfiguration(t *testing.T) {
 		},
 	}
 
-	reqOutputConfigurationAck := []byte{0xfa, 0xff, 0xc1, 0x8, 0x50, 0x40, 0x0, 0x64, 0x20, 0x13, 0x0, 0xc8, 0x49}
```

Comment:
My LSP made some automatic changes here, due to line length. Can revert if you guys want to!

Reply:
Could you make 2 separate commits so its easier to follow what the actual changes are?